### DISCUSSION TITLE
フォントを「ＭＳゴシック」に変更する方法

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1863] [Q&A]
良い一日、

 特定のフォントの英語と日本語の単語を「ＭＳ ゴシック」フォントに変更することは可能ですか？

前もって感謝します。

![image](https://user-images.githubusercontent.com/49851974/190917164-5f83e3b4-88c6-4abc-bfb0-8358432fb3f5.png)
![image](https://user-images.githubusercontent.com/49851974/190958409-afe67580-fbae-4bd1-9dfd-0d0350143ac4.png)

review-custom.sty
![image](https://user-images.githubusercontent.com/49851974/190958435-9d936ce3-9c33-413d-b6eb-a06080fe5c8b.png)

**Comment by kmuto** [comment]
表記のフォント指定方法命令にはlualatexコンパイラが必要です。review-jsbookをお使いだったと思いますが、これはuplatexコンパイラ利用を前提としているので利用不可能です(review-jlreqを使うように最初から作り直せばlualatexコンパイラを利用できますが、苦難はあるでしょう)。
現状で「すべてのゴシック」を「MSゴシックにする」と言い換えられるのであれば、
https://review-knowledge-ja.readthedocs.io/ja/latest/latex/uptex-fonts.html
に示す方法で変更することは可能です。欧文("Gothic Font")の箇所もMSゴシックにする方法も記載しています。
ゴシック全部ではなく「一部の箇所」のみ、となると「できなくはないがすごく大変で、たぶんLuaLaTeXで書き直したほうが早い(そしておそらくRe:VIEWでやるよりTeXでそのまま書いたほうがいいのでは？となる)」がanswerです。
